### ISSUE TITLE
SKGLView on Mac needs a stencil buffer

### ISSUE THREAD

**Issue by mattleibow** [mono/SkiaSharp#189]
This can be seen in the Path Effect sample, the paths don't render. This is because the stencil buffers aren't set up.

**Comment by mattleibow** [comment]
I think this might be an issue with skia: https://bugs.chromium.org/p/skia/issues/detail?id=6025

I tried setting the stencil buffer, but this just adds artifacts:

```csharp
private void Initialize()
{
	WantsBestResolutionOpenGLSurface = true;

	var attrs = new object[]
	{
		//NSOpenGLPixelFormatAttribute.OpenGLProfile, (NSOpenGLPixelFormatAttribute)NSOpenGLProfile.VersionLegacy,
		NSOpenGLPixelFormatAttribute.Accelerated,
		NSOpenGLPixelFormatAttribute.DoubleBuffer,
		NSOpenGLPixelFormatAttribute.Multisample,

		NSOpenGLPixelFormatAttribute.ColorSize, (NSOpenGLPixelFormatAttribute)32,
		NSOpenGLPixelFormatAttribute.AlphaSize, (NSOpenGLPixelFormatAttribute)8,
		NSOpenGLPixelFormatAttribute.DepthSize, (NSOpenGLPixelFormatAttribute)24,
		NSOpenGLPixelFormatAttribute.StencilSize, (NSOpenGLPixelFormatAttribute)8,
		NSOpenGLPixelFormatAttribute.SampleBuffers, (NSOpenGLPixelFormatAttribute)1,
		NSOpenGLPixelFormatAttribute.Samples, (NSOpenGLPixelFormatAttribute)4,
		(NSOpenGLPixelFormatAttribute)0,
	};
	PixelFormat = new NSOpenGLPixelFormat(attrs);
}
```

**Comment by mattleibow** [comment]
As mentioned by the skia guys, it has to do with a dirty stencil buffer.

**Comment by mattleibow** [comment]
Fixed in b53c882f5f84d14699dc13dae7b32c1be2efdf14
